.travis.yml

```
@@ -44,6 +44,13 @@ jobs:
         - julia --project -e 'using Pkg; Pkg.activate("docs"); Pkg.instantiate(); Pkg.develop(PackageSpec(path = pwd()))'
         - julia --project=docs --color=yes docs/make.jl
       after_success: skip
+  include:
+    - name: TOML
+      stage: test
+      os: linux
+      script:
+        - julia -e 'using Pkg; Pkg.develop(PackageSpec(path="ext/TOML")); Pkg.instantiate(); Pkg.test("TOML")'
```

`Pkg.instantiate` is not needed here because (i) everything is stdlibs and (ii) `Pkg.test` does its own resolve anyway. `Pkg.develop` is also not needed, we can just run in the TOML project.
```suggestion
        - julia --project=ext/TOML -e 'using Pkg; Pkg.test()'
```